test/multiplier/store_type.go

```diff
@@ -0,0 +1,68 @@
+package multiplier
+
+import (
+	"github.com/sourcenetwork/corekv/test/action"
+)
+
+func init() {
+	Register(&badger{})
+	Register(&memory{})
+}
```

Comment:
question: What if someone adds both memory and badger in the multiplier?

edit: Trying it seems to only use the first one defined.